plugins/scripts/upload/trns_transform_FASTQ_DNA_Reads_to_KBaseAssembly_PairedEndLibrary.py

```diff
@@ -7,30 +7,44 @@
 import argparse
 import json
 import logging
-
 # 3rd party imports
 # None
 
 # KBase imports
 import biokbase.Transform.script_utils as script_utils
+import subprocess
+import os.path
 
 
 # conversion method that can be called if this module is imported
-def convert(shock_service_url, handle_service_url, input_directory, 
-            object_name, level=logging.INFO, logger=None):
+def transform( shock_service_url, handle_service_url, input_directory, 
+               working_directory, output_file_name,
+               object_name, mean_insert, std_dev, interleaved, read_orientation,
+               sra,
+#level=logging.INFO, 
+             logger=None
+           ):
     """
     Converts FASTQ file to KBaseAssembly.PairedEndLibrary json string.
 
     Args:
         shock_service_url: A url for the KBase SHOCK service.
         handle_service_url: A url for the KBase Handle Service.
         input_directory: Where the FASTQ file can be found.
+        working_directory: The directory the resulting json file will be written to.
+        output_file_name: A file name where the output JSON string should be stored.  
         object_name: A name to use when storing the JSON string.
         mean_insert: The average insert size.
         std_dev: standard deviation of the inserts
         interleaved: Are the reads interleaved?
         read_orientation: Do the reads have an outward orientation?
+        sra:  int flag indicating whether or not to preconvert SRA file into FASTQ
         level: Logging level, defaults to logging.INFO.
+
+    Returns:
+        Something
```

Comment:
For the transform() Returns:, should return either 0 for success or non-zero for failure.